app/presenters/applicant_account_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class ApplicantAccountPresenter
+  def initialize(applicant_bank = nil)
```

Comment:
If no `applicant_bank` is passed in, all the other methods fail. So using `= nil` here moves the errors to the individual methods where the underlying problem (that no `applicant_bank` was passed on initialization) is less obvious.

I'd suggest:
```ruby
  def initialize(applicant_bank)
```

Reply:
Also there is no `applicant_bank` model. Should that be `bank_provider` to maintain consistency.